timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -334,15 +495,12 @@ def fit(  # type: ignore
                 )
                 raise TimeLimitExceeded
 
-        # TODO: disentangle fit_resources computation and validation from tabular logic
-        kwargs = self._preprocess_fit_resources(**kwargs)
-        self.validate_fit_resources(**kwargs)
-
         self._fit(
             train_data=train_data,
             val_data=val_data,
             time_limit=time_limit,
-            **kwargs,
+            verbosity=verbosity,
+            **(self._get_system_resources() | kwargs),
```

Comment:
Can we make this more explicit by replacing `kwargs` in the `fit()` signature with `num_cpus` and `num_gpus`? Currently, the meaning of this line is not very clear and requires checking a few places in the code to understand what's going on.

Reply:
I agree. In a future PR, I'll chop away through all the `**kwargs` in timeseries and see what happens :)